### PR TITLE
User Profile pattern topic update

### PR DESCRIPTION
**Pull request by sdimchevski**
Text link removed and made some updates to the text, also updated the images for profile demo, profile small and "detach from symbol" and added a 2x version

### CODE REVIEW

File: en/patterns/user-profile.md [hunk 1]
```diff
@@ -11,19 +11,18 @@ Use the User Profile Pattern to display user-related information, either as a fu
 <img class="responsive-img" src="../images/profile_demo.png" srcset="../images/profile_demo@2x.png 2x" />
 <img class="responsive-img" src="../images/profile_small.png" srcset="../images/profile_small@2x.png 2x" />
 
-An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar, Text, and Inputs that constitute its layout.
+An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar and Inputs that constitute its layout.
 
 > [!WARNING]
 > After inserting any of the Profiles available as a Pattern you should trigger `Detach from Symbol` in order to be able to generate the Pattern as Angular code. The various Components that lay it out must stay intact and not be detached!
```
Comment (StefanIvanov) [inline en/patterns/user-profile.md:17]:
```suggestion
> After inserting any of the User Profile Patterns you should trigger `Detach from Symbol` in order to be able to generate the Pattern as Angular code. The various Components that lay it out must stay intact and not be detached!
```

File: en/patterns/user-profile.md [hunk 2]
```diff
@@ -11,19 +11,18 @@ Use the User Profile Pattern to display user-related information, either as a fu
 <img class="responsive-img" src="../images/profile_demo.png" srcset="../images/profile_demo@2x.png 2x" />
 <img class="responsive-img" src="../images/profile_small.png" srcset="../images/profile_small@2x.png 2x" />
 
-An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar, Text, and Inputs that constitute its layout.
+An editable variant of the User Profile Pattern, nicely laid out with the appropriate Inputs, is also provided. The User Profile Pattern comes with the styling flexibility provided by the Avatar and Inputs that constitute its layout.
```
Comment (StefanIvanov) [inline en/patterns/user-profile.md:14]: Let's add an image of these editable variants below this paragraph

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Comment by sdimchevski** [comment]
Fixed some small text errors in the _description too

File: en/patterns/user-profile.md (hunk 2 above)
Comment (sdimchevski) [reply en/patterns/user-profile.md:14]: Done. Please have a look

**Review by StefanIvanov — APPROVED** [review]

**Pull request merged**